helm/thingsboard/templates/component-service.yaml

```diff
@@ -44,12 +44,6 @@ spec:
       protocol: {{ $componentValues.port.protocol }}
       name: {{ $componentValues.port.name }}-{{ . }}
     {{- end -}}
-    {{- if $componentValues.http }}
```

Comment:
Needed?

Reply:
yes, we were exposing prometheus metrics in the load balancer, we don't need it for the prometheus scrapper

Reply:
Ok, I see more or less. Can we unset this in values.yaml for mqtt service?

Reply:
It was only added here by me for the mqtt service, all the other services don't have it and were working correctly before I added this part. It was a mistake on my part, it doesn't have any purpose anymore.

Reply:
Ok, thank you for the explanation.